api/api.go

```diff
@@ -324,6 +329,21 @@ func addOrbElementsToBuffer(buf *bytes.Buffer, name string, elems map[string]str
 	}
 }
 
+// String returns a text representation of the Orb contents, intended for
+// direct human use rather than machine use. This function will exclude orb
+// source and orbs without any versions in its returned string.
+func (orb Orb) SimplifiedString() string {
```

Comment:
Same idea here. This is a concern of the CLI (the `cmd`) package, not of the API.